cnv/cnv_users_guide/cnv-ansible-modules.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
```

Comment:
If this file is a module, then it should go in the modules/ directory.

Reply:
Ah, excellent catch! Piotr, please move the file to the modules directory. Thanks :)